sdk/translation/azure-ai-translation-text/samples/README.md

```diff
@@ -0,0 +1,45 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- cognitive-services
+- text-translator
```

Comment:
I think the product slug should be azure-translator-text: https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main#product

Reply:
Changed.